etna/records/views/records.py

```diff
@@ -65,7 +65,7 @@ def record_page_view(request, iaid):
 
     return render(
         request,
-        page.get_template(request),
+        "records/record_page.html",
```

Comment:
Now that the record isn't a page, maybe following regular Django template naming conventions would be a good shout here? (e.g. `records/record_detail.html`)

Reply:
Good point! I've renamed the templates and views for records.